vipps-checkout-api.md

```diff
@@ -161,11 +164,35 @@ sequenceDiagram
   API->>Merchant: Marks payment as succeeded and sends callback
 ```
 
+#### Helthjem integration
+
+**(v3 only)**
+
+Helthjem is a shipping provider that offers delivery to a pickuppoint selected by the customer (and also to the doormat, but there is no integration for that yet). To enable it, the merchant must:
```

Comment:
Do we need to advertise that delivery on the doormat is not supported.
I think we can just leave it out. Especially since it's in parentheses.

Also `To enable it, the merchant must` can maybe be removed or reworded. If you are already reading this, you are maybe attempting to integrate. And you don't have to be a merchant to integrate. You could be an integrator or an IT-partner.